test/unit_tests/braket/experimental/autoqasm/test_converters.py

```diff
@@ -59,3 +59,32 @@ def fn() -> None:
 a = 1;
 e = a;"""
     assert qasm == expected_qasm
+
+
+def test_break_for_loop():
+    @aq.function
+    def main():
+        for i in aq.range(3):
+            aq.gates.h(i)
+            break
+
+    with pytest.raises(aq.errors.UnsupportedFeature):
+        main()
+
+
+def test_break_while_loop():
+    @aq.function
+    def uses_while():
+        while aq.gates.measure(0):
+            aq.gates.x(0)
+
+    uses_while()
```

Comment:
Why do we need to test `while` here?

Reply:
You're right, we don't. I'll remove it, since it's tested elsewhere.

I had originally thought `while` wasn't supported, because it was broken for a bit while I was debugging, then, realizing it was supported, I was suspicious that my implementation was messing with the while functionality, so I added this, but it should be cleaned up now.